tests/manual-test-cases/Group19-DRS-Disabled/19-3-DRS-Disabled-Placement.robot

```diff
@@ -0,0 +1,81 @@
+// Copyright 2018 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+*** Settings ***
+Documentation  Test 19-3 - DRS-Disabled-Placement
+Resource  ../../resources/Util.robot
+# Suite Setup  Wait Until Keyword Succeeds  10x  10m  ROBO SKU Setup
+# Suite Teardown  Run Keyword And Ignore Error  Nimbus Cleanup  ${list}
+
+*** Keywords ***
+Get Host IP
+    [Arguments]  ${name}
+    ${rc}, ${ip}=  Run And Return Rc And Output  govc vm.info ${name} |grep Host |awk '{print $2}'
+    Should Be Equal  ${rc}  0
+    Return  ${ip}
+
+Get VM Name
+    [Arguments]  ${name}  ${id}
+    ${id_prefix}=  Get Substring  ${id}  0  12
+    ${vm_name}=  Catenate  SEPARATOR=_  ${name}  ${id_prefix}
+    Return ${vm_name}
+
+Deploy Stress Container And Return Host IP
+    [Arguments]  ${name}
+    ${rc} ${id}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -d --name ${name} progrium/stress --vm 5 --vm-bytes 1000M --vm-hang 0
```

Comment:
`--vm-bytes` needs to be something like `256M` otherwise the containerVM won't boot.